internal/pkg/pkgfile/build.go

```diff
@@ -143,7 +147,7 @@ func Build(ctx context.Context, c client.Client, options *environment.Options) (
 				Definition: def.ToPB(),
 			})
 			if err != nil {
-				return fmt.Errorf("failed to resolve dockerfile: %q", err)
+				return fmt.Errorf("failed to resolve dockerfile: %w", err)
```

Comment:
```suggestion
				return fmt.Errorf("failed to resolve %s: %w", constants.PkgFile, err)
```

Reply:
it looks like this error is actually about solving the whole LLB, so it was completely inaccurate